Getting-Started/Setup/Server-Setup/azure-web-apps.md

```diff
@@ -62,15 +62,17 @@ The `SyncTempEnvDirectoryFactory` enables Examine to sync indexes between the re
 * If you have a {machinename} token in your `~/Config/ExamineIndex.config` file remove this part of the path. Example, if you have path that looks like: `~/App_Data/TEMP/ExamineIndexes/{machinename}/External/` it should be `~/App_Data/TEMP/ExamineIndexes/External/` 
 * Due to the nature of Lucene files and IO latency, you should update all of your Indexers and Searchers in the `~/Config/ExamineSettings.config` file to have these two properties (see [here](http://issues.umbraco.org/issue/U4-7614) for more details): `useTempStorage="Sync"`
 
-### Umbraco XML cache file
+### Umbraco XML cache file and other TEMP files
 
 For a single Azure Web App instance you need to ensure that the Umbraco XML config file is stored on the local server (since Azure uses a shared file system). To do this you need to add a new app setting to web.config:
 
 For **Umbraco v7.7.3+**
 
+It is recommended that you upgrade your Azure Web App to support this setting to avoid locking issues with TEMP files during server migrations or slot swapping. See [U4-10503](http://issues.umbraco.org/issue/U4-10503) for more information on this.
```

Comment:
what do you mean with "support your Azure Web App"?

Reply:
Good point, how about “upgrade your Azure Web App hosted Umbraco”

Reply:
I have rephrased the whole paragraph now